products/digital-sales-rooms/installation/realtime-service-mercure.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+   title: Realtime Service - Mercure
+   position: 50
+
+---
+
+# Realtime Service - Mercure
+
+::: info
+Symfony provides a straightforward component, built on top of the [Mercure](https://symfony.com/doc/current/mercure.html) protocol, specifically designed for this class of use cases.
+[Mercure](https://mercure.rocks/docs/getting-started) is an open protocol designed from the ground up to publish updates from server to client. It is a modern and efficient alternative to timer-based polling and to WebSocket.
+:::
+
+## Mercure general settings
+Detailed below is the minimum configuration needed for a working stack apart from project-specific CMS configurations.
+- Set up CORS allowed origins - In our case, it would be the domain where the Shopware Frontends is hosted and available. For instance: https://dsr-frontends.com (frontend).
+- Set up publish allowed origins - The domains that request the Mercure service must be added to publish allowed origins or else it gets rejected. For instance (HTTP protocol must not be included): shopware-pwa-gs.herokuapp.com (frontend) and pwa-demo-api.shopware.com(backend - API).
+- Set up the publisher (JWT) key - Set whatever you want.
+- Set up the subscriber (JWT) key - Set whatever you want.
```

Comment:
```suggestion
- *Set up CORS allowed origins* - In our case, it would be the domain where the Shopware Frontends is hosted and available. For instance: https://dsr-frontends.com (frontend).
- *Set up publish allowed origins* - The domains that request the Mercure service must be added to publish allowed origins or else it gets rejected. For instance (HTTP protocol must not be included): shopware-pwa-gs.herokuapp.com (frontend) and pwa-demo-api.shopware.com(backend - API).
- *Set up the publisher (JWT) key* - Set whatever you want.
- *Set up the subscriber (JWT) key* - Set whatever you want.
```